.travis.yml

```diff
@@ -153,7 +153,6 @@ jobs:
         - travis_wait make test-cmd-docker-devfile-push
         - travis_wait make test-cmd-docker-devfile-url
 
-    
```

Comment:
Keep the travis file like it was. Adding a space or removing new line does make any sense. I guess it has been made accidentally, please revert it back.

Reply:
Yes, I got removed accidently. Good catch 👍